tests/test_downloader_handlers.py

```diff
@@ -493,7 +493,10 @@ def test_download_broken_chunked_content_allow_data_loss_via_setting(self):
 class Https11TestCase(Http11TestCase):
     scheme = 'https'
 
-    tls_log_message = 'SSL connection certificate: issuer "/C=IE/O=Scrapy/CN=localhost", subject "/C=IE/O=Scrapy/CN=localhost"'
+    tls_log_message = (
+        'SSL connection certificate: issuer "/C=IE/O=Scrapy/CN=localhost", '
+        'subject "/C=IE/O=Scrapy/CN=localhost"'
```

Comment:
Almost all changes in this PR are for good, but splitting constants like that is why I don't like strict line length restrictions. Here the wrapping makes it easier to make an error (+- a whitespace), and harder to understand what's in the constant. Nothing critical, but I wonder if for such lines we can ignore the warning, instead of changing the code.

Reply:
Makes sense, yes. I'm gonna go with `# noqa` in the the specific lines, I feel like keeping the files in `pytest.ini` is too much of a carte blanche.